### DISCUSSION TITLE
Third-party launcher (Nova Lanucher) can't create a shortcut for the parasitic manager

### DISCUSSION THREAD

**Discussion by Saeviome** [LSPosed/LSPosed#1573] [Q&A] [answered]
Shortcuts can be created by MIUI default launcher .
After installing the real manager, click the button in the floating window, the shortcut cannot be created .

Chinese translation :
第三方启动器（Nova启动器）无法创建寄生管理器的快捷方式。
MIUI官方桌面可以创建快捷方式。
安装真实管理器后，点击悬浮窗中的按钮，无法创建快捷方式。

-----------OS & Runtime-----------
MIUI 12.5.1 @Android 10
Magisk 23.0
LSPosed 1.65

**Comment by yujincheng08** [comment, marked as answer]
#1497

**Reply by Saeviome** [reply]
The solution, after deleting the shortcut from the launcher that has created a parasitic shortcut, you can create a new one on the third-party launcher.

Chinese translation :
解决方案，在已建立寄生快捷方式的启动器里删除快捷方式后，就可以在第三方启动器新建了。

**Reply by Hr46ph** [reply]
Can you please elaborate because I dont understand what you mean. I need to keep the manager installed because its not creating shortcuts to the parasitic manager.

With Magisk, it pops up an icon that can be dragged and it will show the default launcher (nova in my case) homescreen to drop it. LSposed manager doesn't show that and automatically creates it, which doesn't work when Nova is the default launcher.

I was going to ask support on Nova btw when I found this.

Thanks.

**Reply by Hr46ph** [reply]
I had installed the manager from /data/adb/etc, which doesn't popup the icon to drag and drop. After updating to 1.7.1, there was an icon popup for the shortcut.

Solved  :).

**Comment by joeyoropesa-dev** [comment]
It's not solved for me - I'm using the latest LSPosed version in 2023 and I deleted LSPosed Shortcut due to the launcher bug changes so I was forced to re-create all shortcuts from the beginning.

How can I do that **without reinstalling LSposed Module** - what command I need to enter via shell or etc... because the question about creating a new shortcut doesn't appearing in the default LSPosed application I've installed from /data/adb/lspd
